packages/atomic/CHANGELOG.md

```diff
@@ -7,1794 +7,939 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
```

Comment:
> That's a tough PR to review with 153 file changes...
> 
> It modifies a bunch of unrelated files across the whole repo, that gets auto generated by different tooling (stencil, lerna, etc.) We don't want that "linted" or "prettified".
> 
> If it stayed in the `quantic` sub packages I would tell you "go ahead do what you want", but as it stands, I don't have the mental power to sift through all the changes to make sure everything is okay.
> 
> Also, FYI, we've had many problems with the linting changes that were merged many months ago: people unable to commit, build failing on CI due to random generated test files getting flagged as "incorrect", release process getting broken, etc.
> 
> I really do not want this to repeat because it was quite a waste of time to fix... 😿

I understand, but on the other hand, what should I do then when I merge main/master in a branch that's solely related to quantic and I have stuff from atomic getting formatted because for some reason it hasn't been?.